streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBWindowStoreTest.java

```diff
@@ -98,14 +99,25 @@
     private WindowStore windowStore;
 
     @SuppressWarnings("unchecked")
-    private <K, V> WindowStore<K, V> createWindowStore(ProcessorContext context, final boolean enableCaching, final boolean retainDuplicates) {
-        final RocksDBWindowStoreSupplier supplier = new RocksDBWindowStoreSupplier<>(windowName, retentionPeriod, numSegments, retainDuplicates, Serdes.Integer(), Serdes.String(),
-                                                                                     WINDOW_SIZE, true, Collections.<String, String>emptyMap(), enableCaching);
-        final WindowStore<K, V> store = (WindowStore<K, V>) supplier.get();
+    private WindowStore createWindowStore(ProcessorContext context, boolean retainDuplicates) {
```

Comment:
add missing generics to return type  and remove "unchecked" suppression?

nit: add `final` to parameters